pinc/ProjectTransition.inc

```diff
@@ -1168,11 +1184,76 @@ new ProjectTransition(
         'action_name'           => _("Return to Post-Processor"),
         'confirmation_question' => _("Are you sure you want to return this book to the post-processor for further work?"),
         'detour'                => null,
-        'settings_template'     => "modifieddate='<TIMESTAMP>', checkedoutby=postproofer, postproofer=''",
-        'collateral_actions'    => null,
+        // formerly ppverifier was only set when the project was uploaded to PG
+        // it is set here so the PPer can return it to the same PPVer
+        // any transition out of PROJ_POST_FIRST_CHECKED_OUT sets it to NULL
+        // this will not affect how the PPVer is ultimately set.
```

Comment:
I find this comment confusing. Do you mean "If the PPer chooses to return the project to the PP pool after the PPVer returns it to their PP queue, the `ppverifier` field is set to `NULL`."? Is "this will not affect how the PPVer is ultimately set." relevant here?

Reply:
I was trying to change from the way it works now as little as possible. In addition to the case of returning to the PP pool you mentioned it also gets set to NULL when the project goes back to the PPVer. This is how it would have been previously although it now seems illogical and is not necessary.
The comment about how the PPVer is ultimately set was meant to indicate that I had not overlooked the cases where the ultimate PPVer could be a different person or none if this PPVer returns the project to the PPV pool (or it subsequently goes via the PPer to the PP pool or to "posted to PG" if the PPer aquires DU in the meantime).

Reply:
I think what confused me was that you said `any transition out of PROJ_POST_FIRST_CHECKED_OUT`, and I was just thinking that meant that it was being returned by the PPer to the PP pool. I was forgetting that it's equally true that returning it to the PPVer is a transition out of `PROJ_POST_FIRST_CHECKED_OUT`. Thanks for the clarification.